frontend/src/routes/project/[project_code]/+page.ts

```diff
@@ -65,6 +66,58 @@ export async function _addProjectUser(input: AddProjectMemberInput) {
     return result;
 }
 
+export async function _changeProjectName(input: ChangeProjectNameInput) {
+    //language=GraphQL
+    const result = await getClient().mutation(
+        graphql(`
+            mutation ChangeProjectName($input: ChangeProjectNameInput!) {
+                changeProjectName(input: $input) {
+                    project {
+                        id
```

Comment:
we should try to make a habit of selecting out the data that was updated in a mutation (here and on the change description mutation), I believe we can remove the manual invalidation we're doing below in that case, but I'm not sure, either way it's good practice in gql to try to select out the data that get's changed along with the id so the gql client can update the cached data automatically. This doesn't work so well in the case of removing something from a list like in the members example below however.